mautrix_signal/commands/signal.py

```diff
@@ -81,15 +81,16 @@ async def pm(evt: CommandEvent) -> None:
 
 @command_handler(needs_auth=True, management_only=False, help_section=SECTION_SIGNAL,
                  help_text="View the safety number of a specific user",
-                 help_args="[--qr] [_phone_]")
+                 help_args="[--qr] <_phone_>")
```

Comment:
This was the wrong way initially, so now it should actually be square brackets

```suggestion
                 help_args="[--qr] [_phone_]")
```